tableau_utilities/hyper/hyper.py

```diff
@@ -0,0 +1,112 @@
+import os
+import shutil
+from zipfile import ZipFile
+from tableauhyperapi import HyperProcess, Connection, Telemetry, CreateMode, TableDefinition, TableName, SqlType
+
+from tableau_utilities.tableau_file.tableau_file import TableauFileError
+
+
+def empty_hyper_extract(Datasource):
```

Comment:
Can you add the action this function is doing in the alias?
```suggestion
def create_empty_hyper_extract(datasource: Datasource):
```
Also, you should use lowercase here for function arguments, by convention.
You can also suggest the type it should expect, as the Datasource class.
Same goes for the below function.

You'll need to import the Datasource object if you want to do this type suggestion.